src/features/community/scientist/components/Incubator.tsx

```diff
@@ -272,7 +271,7 @@ export const Incubator: React.FC = () => {
           <div className="flex flex-col-reverse sm:flex-row">
             <div
               className="w-full sm:w-3/5 h-fit overflow-y-auto scrollable overflow-x-hidden p-1 mt-1 sm:mt-0 sm:mr-1 flex flex-wrap"
-              style={{ maxHeight: TAB_CONTENT_HEIGHT }}
+              style={{ maxHeight: 400 }}
```

Comment:
Any specific reason why you removed the constant name for this? I really prefer giving context to magic numbers especially when they're used consistently in many components and are necessary to keep the style consistent.

Reply:
I am planning to phase out TAB_CONTENT_HEIGHT and use the new panel component constants.  TAB_CONTENT_HEIGHT was removed from the refactored files so I temporary set them to 400 for all the existing references.  Eventually these will be deleted in future refactorings